accelerator/core/serializer/serializer.c

```diff
@@ -393,7 +393,22 @@ static status_t transaction_array_to_json_array(cJSON* json_root, const transact
 
     ret = iota_transaction_to_json_object(txn, &txn_obj);
     if (ret != SC_OK) {
-      ta_log_error("%s\n", error_2_string(ret));
+      ta_log_error("%s\n", ta_error_to_string(ret));
+      return ret;
+    }
+    cJSON_AddItemToArray(json_root, txn_obj);
+  }
+  return ret;
+}
+
+static status_t bundle_transaction_to_json_array(const bundle_transactions_t* const bundle, cJSON* json_root) {
+  status_t ret = SC_OK;
+  iota_transaction_t* txn = NULL;
+  BUNDLE_FOREACH(bundle, txn) {
+    cJSON* txn_obj = cJSON_CreateObject();
```

Comment:
Delete `txn_obj` when no further use it.

Reply:
We will release the whole object instead of a single element.

Reply:
When `ret != SC_OK` in line 410, `txn_obj` won't join `json_root`

Reply:
Where do you solve this?

Reply:
here https://github.com/DLTcollab/tangle-accelerator/pull/590/files#diff-ce8a4d1e935b3c0d008fd36f3d9857fdR401

Reply:
The link can not work.
And I do not mean to release the whole object.

Reply:
If you mean the release in `transaction_array_to_json_array`.
That is another obj in another function.